### PR TITLE
[BF] find_cuts crashes ungracefully if mask is empty

### PR DESCRIPTION
**Pull request by dohmatob**
Fixes issue #855

### CODE REVIEW

File: nilearn/plotting/find_cuts.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
     data = as_ndarray(data)
     my_map = data.copy()
     if mask is not None:
+        # check against empty mask
+        if mask.sum() == 0.:
+            raise ValueError("Provided mask is empty!")
```
Comment (GaelVaroquaux) [inline nilearn/plotting/find_cuts.py]: Maybe another option, possibly more useful for the end user, would be to raise this as a warning, and return the geometrical center of the mask.

Comment (dohmatob) [inline nilearn/plotting/find_cuts.py]: Gr8 suggestion.

Comment (GaelVaroquaux) [inline nilearn/plotting/find_cuts.py]: Hey @dohmatob : any chance that you could implement the suggestion, so that we merge this PR.

**Comment by dohmatob** [comment]
I've just implemented the suggestion at the moment.

**Comment by lesteve** [comment]
Looks fine, merging, thanks!

**Comment by lesteve** [comment]
For the record "fix issue #bla" doesn't seem to close automatically the issue once the PR is merged...

**Pull request merged**